spaces-config-service-api/src/main/proto/org/hypertrace/spaces/config/service/v1/spaces_config_service.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+
+package org.hypertrace.spaces.config.service.v1;
+
+import "org/hypertrace/spaces/config/service/v1/space_config_rule.proto";
+
+service SpacesConfigService {
+
+  rpc CreateRule (CreateRuleRequest) returns (CreateRuleResponse) {
+  }
+  rpc GetRules (GetRulesRequest) returns (GetRulesResponse) {
+  }
+  rpc UpdateRule (UpdateRuleRequest) returns (UpdateRuleResponse) {
+  }
+  rpc DeleteRule (DeleteRuleRequest) returns (DeleteRuleResponse) {
+  }
+}
+
+message CreateRuleRequest {
+  oneof rule_data {
+    AttributeValueData attribute_value_data = 1;
+  }
+}
+
+message CreateRuleResponse {
+  SpaceConfigRule rule = 1;
+}
+
+message GetRulesRequest {
```

Comment:
I don't have much context here, but what if there are so many rules, you want to get either few of them or only one rule

Reply:
We don't currently have a use case where we don't need all rules. Also, for now the expected number of rules is between 0 and 1 - as the feature is built out further for more use cases, we will revisit this.

The two current use cases are
- the space enrichment evaluation, in which case all rules must be evaluated
- The space configuration, in which case all rules are displayed in a table (currently no paging UX for that)